### DISCUSSION TITLE
I want to be able to determine when a particular satellite is above some altitude but some others are not

### DISCUSSION THREAD

**Discussion by jgrahamc** [skyfielders/python-skyfield#888] [Q&A]
Specifically, I want to to be able to determine when the Prospero X-3 satellite is above x degrees altitude from my location, but satellites owned by Orbcomm are not. The reason is they share the same downlink frequency and I am trying to pick up the Prospero X-3 signal without the QRM from the Orbcomm fleet. I have the TLEs for all of them. Anyone written something using Skyfield to compute "X is up but A B C are not"?

**Comment by ngc1535git** [comment]
Actually... in a way, I have done this. I have written a satellite scheduler. It computes and graphically shows the visibility of satellites based on constraints (useful for creating an observing program). It was not designed to precisely answer your question... but it would return plots of visibility (like a doodle poll) for any list of satellites. So if you gave me the norad ID of Prospero and the other satellites- it would print out a plot like the below. In your case, you would look for times/nights when Prespero was visible and no other satellites had lines at those times. 
![monetgrams_schedule](https://github.com/skyfielders/python-skyfield/assets/47374461/5befaea7-5efb-4942-abb7-773bf6a17fbe)
